superbench/common/utils/__init__.py

```diff
@@ -10,11 +10,6 @@
 nv_helper = LazyImport('superbench.common.utils.nvidia_helper')
 
 __all__ = [
-    'LazyImport',
-    'SuperBenchLogger',
-    'create_sb_output_dir',
-    'get_sb_config',
-    'logger',
-    'nv_helper',
-    'rotate_dir',
+    'LazyImport', 'SuperBenchLogger', 'create_sb_output_dir', 'get_sb_config', 'logger', 'nv_helper', 'rotate_dir',
+    'network'
```

Comment:
add trailing comma
sort in alphabet order

Reply:
thank you